src/js/themes/media-theme-micro.js

```diff
@@ -0,0 +1,538 @@
+    overflow: hidden;
+    width: 0;
+  }
+
+  .volume-group:hover .volume-range-span,
+  [media-keyboard-control] .volume-group:focus-within .volume-range-span {
+    width: var(--_volume-range-expand-width);
+    padding-left: var(--_volume-range-padding-left);
+    padding-top: var(--_volume-range-padding-top);
+  }
+
+  :host([control-bar-vertical]) .volume-range-span {
+    display: inline-flex;
+    height: 0;
+    --_volume-range-padding-left: 0 !important;
+  }
+
+  :host([control-bar-vertical]) .volume-group:hover .volume-range-span,
+  :host([control-bar-vertical]) [media-keyboard-control] .volume-group:focus-within .volume-range-span {
+    width: auto;
+    max-width: 40px;
+    height: var(--_volume-range-expand-height);
+  }
+
+  media-volume-range {
+    --media-range-thumb-opacity: 0;
+    --media-range-track-border-radius: 2px;
+    --media-range-track-background: rgba(255, 255, 255, .2);
+    --media-range-bar-color: rgba(255, 255, 255, .9);
+
+    width: var(--_volume-range-expand-width);
+    display: var(--controls, var(--volume-range, inline-block));
+    border-radius: 5px;
+  }
+
+  :host([control-bar-vertical]) media-volume-range {
+    width: var(--_volume-range-expand-height);
+    transform: rotate(-90deg);
+  }
+
+  media-control-bar:has(.volume-group:hover),
+  [media-keyboard-control] media-control-bar:has(.volume-group:focus-within) {
+    top: var(--_control-bar-offset-top, 0);
+    left: var(--_control-bar-offset-left, calc(var(--_volume-range-expand-width) / 2));
+  }
+
+  :host([control-bar-place$="start"]),
+  :host([control-bar-place$="end"]) {
+    --_control-bar-offset-left: 0;
+  }
+
+  :host([control-bar-vertical]) {
+    --_control-bar-offset-left: 0;
+  }
+
+  :host([control-bar-vertical][control-bar-place^="center"]) {
+    --_control-bar-offset-top: calc(var(--_volume-range-expand-height) / 2);
+  }
+
+  media-time-range {
+    --media-range-track-height: 6px;
+    --media-range-track-transition: height .1s linear;
+    --media-range-track-background: rgba(20, 20, 30, .25);
+    --media-time-buffered-color: rgba(20, 20, 30, .3);
+    --media-range-bar-color: rgba(255, 255, 255, .9);
+    --media-range-track-box-shadow: 0 -1px 0 rgba(20, 20, 30, .07);
+    --media-range-thumb-opacity: 0;
+    --media-range-padding-left: 0;
+    --media-range-padding-right: 0;
+    --media-preview-time-background: var(--_secondary-color);
+    --media-preview-time-margin: 0 0 8px;
+
+    display: var(--controls, var(--time-range, inline-block));
+    width: 100%;
+    height: 10px;
+    bottom: -2px;
+  }
+
+  media-live-button {
+    --media-control-background: var(--_secondary-color);
+    --media-control-hover-background: var(--_secondary-color);
+    border-radius: 5px;
+  }
+
+  media-live-button::before {
+    content: '';
+    width: 5px;
+  }
+
+  media-live-button::after {
+    content: '';
+    width: 7px;
+  }
+
+  media-play-button {
+    display: var(--controls, var(--play-button, inline-flex));
```

Comment:
suggestion(non-blocking): We may want to consider adding `--bottom-controls`, `--bottom-play-button`, etc. for predictability when folks swap out themes (not a high priority)